packages/manager/apps/vrack-services/src/data/vrack-services/vs-api-utils.ts

```diff
@@ -40,6 +40,13 @@ export const useVrackServicesList = (refetchIntervalTime = 5000) => {
   });
 };
 
+export const REGEX =
+  '^(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?).0/24$';
+// The subnet address is limited to only "/24".
+export const isValidCidr = (subnet: string): boolean => {
```

Comment:
Maybe you should add unit tests on this to make it understandable

Reply:
As of now we dont have unit testing on vrack services. We are aiming after beta2 launch.